src/main/java/it/gov/pagopa/cgn/portal/facade/DocumentFacade.java

```diff
@@ -20,6 +24,16 @@
     private final DocumentService documentService;
     private final DocumentConverter documentConverter;
 
+    public ResponseEntity<Resource> getDocumentTemplate(String agreementId, String documentType) {
+        byte[] document = documentService.renderDocument(agreementId, DocumentTypeEnum.valueOf(documentType.toUpperCase())).toByteArray();
+
+        return ResponseEntity
+                .ok()
+                .contentLength(document.length)
+                .contentType( MediaType.APPLICATION_PDF)
```

Comment:
```suggestion
                .contentType(MediaType.APPLICATION_PDF)
```

Reply:
LGTM anyway. Once we correct indentation we can merge :)

Reply:
Done, thanks